### PR TITLE
Add support to enable single ingress check

### PR DESCRIPTION
**Pull request by songrx1997**
/assign @swetharepakula

### CODE REVIEW

File: cmd/check-gke-ingress/app/command/root.go [hunk 1]
```diff
@@ -48,7 +48,14 @@ var rootCmd = &cobra.Command{
...
 		}
-		output := ingress.CheckAllIngresses(namespace, client, beconfigClient, feConfigClient)
+
+		var output report.Report
+		if len(args) == 0 {
+			output = ingress.Check("", namespace, client, beconfigClient, feConfigClient)
+		} else {
+			output = ingress.Check(args[0], namespace, client, beconfigClient, feConfigClient)
+		}
+
```
Comment (swetharepakula) [inline cmd/check-gke-ingress/app/command/root.go:58]: This check is repeated again in ingress.Check when you verify whether the ingress name is empty or not. Split into ingress.Check and ingress.CheckAllIngresses.

Then rename the the existing CheckAllIngress to something like RunChecks

Comment (songrx1997) [reply cmd/check-gke-ingress/app/command/root.go]: Done, thanks!

**Review by swetharepakula — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**